Framework/jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/SeleniumConfig.java

```diff
@@ -304,200 +197,62 @@ public static WebDriver getRemoteBrowser(String remoteBrowser) throws Exception
    * @return The platform (or OS) to run remote tests against
    */
   public static String getRemotePlatform() {
-    return Config.getValueForSection(SELENIUM_SECTION, "RemotePlatform");
+    return Config.getValueForSection(SELENIUM_CAPS_SECTION, "platform");
```

Comment:
MAQS uses the standard Selenium section for this instead of the remote caps section. Should we change our config to match?

This applies to getRemoteBrowserName(), getRemoteBrowserVersion() as well.

Reply:
I think this should be discussed as a group - I wasn't sure which section to use